cli/src/test/scala/com/lightbend/rp/reactivecli/runtime/kubernetes/KubernetesPackageTest.scala

```diff
@@ -302,11 +227,11 @@ object KubernetesPackageTest extends TestSuite {
                 |	}
                 |}
               """.stripMargin.parse.right.get
-
+            println(ingress.payload)
```

Comment:
I assume this was committed by accident so I'll remove it.

Reply:
Thanks Jason 😳